tests/TestCase/LoginTrait.php

```diff
@@ -0,0 +1,53 @@
+<?php
+namespace App\Test\TestCase;
+
+use Cake\Core\Configure;
+
+/**
+ * FoodCoopShop - The open source software for your foodcoop
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @since         FoodCoopShop 3.0.3
```

Comment:
the next stable release will be 3.1.0
3.0.3 (not sure if it will be released) won't get that new file